crates/telio-utils/src/local_interfaces.rs

```diff
@@ -0,0 +1,72 @@
+#[cfg(feature = "test-util")]
```

Comment:
Try to avoid having this conditional compilation and use proper dev build dependencies